documentation/modules/auxiliary/server/socks4a.md

```diff
@@ -0,0 +1,60 @@
+
+## Verification Steps
+
+  1. Start msfconsole
+  2. Do: ```use auxiliary/server/socks4a```
+  3. Do: ```run```
+  4. Do: ```curl --proxy socks4a://localhost:1080 https://github.com```
```

Comment:
You can use ```-x```, but I'm just being nitpicky like https://github.com/rapid7/metasploit-framework/pull/7340/files#r79523680 to get my comment count up. Your way is clearer. :D